include/sensors.h

```diff
@@ -12,27 +12,30 @@ extern "C" {
 extern vector_t _accel;
 extern vector_t _gyro;
 extern float _imu_temperature;
-extern uint32_t _imu_time;
-extern bool _imu_ready;
+extern uint64_t _imu_time;
 
 extern bool _diff_pressure_present;
```

Comment:
Or rename the data fields to `_diff_pressure`, `_diff_pressure_velocity`, `_diff_pressure_temp`? That would probably be cleaner